airmozilla/main/static/main/js/calendar.js

```diff
@@ -2,6 +2,10 @@ $(function() {
     var container = $('#calendar');
     var startsOnMonday = $('#startsOnMonday');
     var storage = localStorage.getItem('startsOnMonday');
+    var view = localStorage.getItem('view');
+    var save_view = function() {
```

Comment:
camelCase is the norm here. Eg. `startsOnMonday`, `initCalendar` etc. 
So then it should be `saveView`.